R/app.R

```diff
@@ -96,6 +97,11 @@ shinyApp <- function(ui, server, onStart=NULL, options=list(),
   shinyOptions(appDir = getwd())
   appOptions <- consumeAppOptions()
 
+  # Also store `lang` option to be passed for rendering processors.
+  if(!is.null(lang) && is.character(lang)) {
```

Comment:
Can `lang` be anything other than a string? If not, then when `is.character(lang)`, we should make sure it's length 1.